config/chain/sei-common.ts

```
@@ -0,0 +1,27 @@
+import { CHAIN_TYPE, NETWORK } from '../schemas/common.schema.js';
+import { getCommonChainInput, SEI_L1_CONTRACT_ADDRESSES, SEI_L2_TOKEN_ADDRESSES } from './common.chain.js';
```

ESLint

[error] Replace `·getCommonChainInput,·SEI_L1_CONTRACT_ADDRESSES,·SEI_L2_TOKEN_ADDRESSES·` with `⏎··getCommonChainInput,⏎··SEI_L1_CONTRACT_ADDRESSES,⏎··SEI_L2_TOKEN_ADDRESSES,⏎`

(prettier/prettier)